e2e/tests/cluster_restart.robot

```diff
@@ -15,27 +17,27 @@ ${CONTROL_PLANE_NODE_NETWORK_LATENCY_IN_MS}    0
 
 *** Test Cases ***
 Restart Cluster While Workload Heavy Writing
-    Given Create deployment 0 with rwo volume
-    And Create deployment 1 with rwx volume
-    And Create deployment 2 with rwo and strict-local volume
-    And Create statefulset 0 with rwo volume
-    And Create statefulset 1 with rwx volume
-    And Create statefulset 2 with rwo and strict-local volume
+    Given Create deployment deploy-0 using rwo volume
+    And Create deployment deploy-1 using rwx volume
+    And Create deployment deploy-2 using rwo volume created from strict-local storageclass
```

Comment:
Could you guide us what's the test step would look like if it wants to create a volume with arbitrary options? Like `numberOfReplicas`, `migratable`, `fromBackup`, or `encrypted`.

Especially when it has multiple the above options at the same time. How can we better describe it?

Reply:
This is another topic for improvement. We had briefly discussed [here](https://github.com/longhorn/longhorn-tests/pull/1300#discussion_r1274344624). I believe introducing an intermediate step to manage manifest customization would be more effective than the current template approach.

Let me create an issue for this.

Reply:
https://github.com/longhorn/longhorn/issues/7223